lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,49 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+
```

Comment:
[Whitespace directly after the method declaration](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods) is unruby-like.  Please remove this empty line.